bin/snap-sync.bash

```diff
@@ -70,33 +70,33 @@ traperror() {
     printf "exit status: %s\n" "$2"
     printf "command: %s\n" "$3"
     printf "bash line: %s\n" "$4"
-    printf "function name: %s\n" "$5"
+    printf "function progname: %s\n" "$5"
     exit 1
 }
 
-trapkill() { 
-    die "Exited due to user intervention." 
+trapkill() {
+    die "Exited due to user intervention."
 }
 
-trap 'traperror ${LINENO} $? "$BASH_COMMAND" $BASH_LINENO "${FUNCNAME[@]}"' ERR
+trap 'traperror ${LINENO} $? "$BASH_COMMAND" $BASH_LINENO "${FUNCPROGNAME[@]}"' ERR
 trap trapkill SIGTERM SIGINT
 
 usage() {
   cat <<EOF
-$name $version
-Usage: $name [options]
+$progname $version
+Usage: $progname [options]
 
 Options:
  -d, --description <desc> Change the snapper description. Default: "latest incremental backup"
  -c, --config <config>    Specify the snapper configuration to use. Otherwise will perform for each snapper
                           configuration. Can list multiple configurations within quotes, space-separated
                           (e.g. -c "root home").
  -n, --noconfirm          Do not ask for confirmation for each configuration. Will still prompt for backup
-                          directory name on first backup"
```

Comment:
It looks like you did a search and replace which changes "name" to progname". Some of these should not have been changed.